vochain/indexer/process.go

```diff
@@ -202,6 +207,7 @@ func (idx *Indexer) newEmptyProcess(pid []byte) error {
 		EntityID:          nonNullBytes(eid),
 		StartBlock:        int64(p.StartBlock),
 		EndBlock:          int64(p.BlockCount + p.StartBlock),
```

Comment:
should we still insert EndBlock when creating a new process? it's StartBlock+BlockCount, so until the process is actually ended (either automatically or manually), it's not really useful at all.

perhaps it's best to always have it there for consistency, but wanted to flag this anyway.

Reply:
yes, for consistency, it makes sense to set it up when creating the process. Thanks for flagging.